WeakAuras/WeakAuras.lua

```diff
@@ -5703,7 +5734,7 @@ function WeakAuras.SetModel(frame, model_path, model_fileId, isUnit, isDisplayIn
   elseif isUnit then
     pcall(function() frame:SetUnit(data) end)
   else
-    if WoW82 then
+    if WoW82 or WeakAuras.IsClassic then
```

Comment:
Aren't models unused in WeakAuras classic? If so, why not just have this function not exist in classic?